.theia/tasks.json

```diff
@@ -0,0 +1,73 @@
+{
+    // Some sample Theia development tasks
+    "tasks": [
```

Comment:
I like having a sample tasks.json but maybe the sample config should be in the package README? Or the supplied sample tasks should be commented out? I'm not sure master should have default tasks when starting theia. Maybe having a package README which would describe a sample task and go in details about every attributes? Or simply a link to vscode's page on the task.json file attributes if they're very similar.

Reply:
Good suggestions. I removed the example tasks.json and added a README, in which I list the example tasks.